docs/conf.py

```diff
@@ -145,3 +145,5 @@
 rst_epilog = '\n'.join(map(lambda x: f".. |{x}| replace:: {frozen_locals[x]}", variables_to_export))
 del frozen_locals
 
+def setup(app):
+    app.add_js_file('../_static/js/piwik.js')
```

Comment:
This is not how you add js files in sphinx:
https://docs.readthedocs.io/en/stable/guides/adding-custom-css.html